src/acl.c

```diff
@@ -61,27 +61,27 @@ struct ACLCategoryItem {
     const char *name;
     uint64_t flag;
 } ACLCommandCategories[] = { /* See redis.conf for details on each category. */
-    {"keyspace", CMD_CATEGORY_KEYSPACE},
-    {"read", CMD_CATEGORY_READ},
-    {"write", CMD_CATEGORY_WRITE},
-    {"set", CMD_CATEGORY_SET},
-    {"sortedset", CMD_CATEGORY_SORTEDSET},
-    {"list", CMD_CATEGORY_LIST},
-    {"hash", CMD_CATEGORY_HASH},
-    {"string", CMD_CATEGORY_STRING},
-    {"bitmap", CMD_CATEGORY_BITMAP},
-    {"hyperloglog", CMD_CATEGORY_HYPERLOGLOG},
-    {"geo", CMD_CATEGORY_GEO},
-    {"stream", CMD_CATEGORY_STREAM},
-    {"pubsub", CMD_CATEGORY_PUBSUB},
-    {"admin", CMD_CATEGORY_ADMIN},
-    {"fast", CMD_CATEGORY_FAST},
-    {"slow", CMD_CATEGORY_SLOW},
-    {"blocking", CMD_CATEGORY_BLOCKING},
-    {"dangerous", CMD_CATEGORY_DANGEROUS},
-    {"connection", CMD_CATEGORY_CONNECTION},
-    {"transaction", CMD_CATEGORY_TRANSACTION},
-    {"scripting", CMD_CATEGORY_SCRIPTING},
+    {"keyspace", ACL_CATEGORY_KEYSPACE},
+    {"read", ACL_CATEGORY_READ},
+    {"write", ACL_CATEGORY_WRITE},
+    {"set", ACL_CATEGORY_SET},
+    {"sortedset", ACL_CATEGORY_SORTEDSET},
+    {"list", ACL_CATEGORY_LIST},
+    {"hash", ACL_CATEGORY_HASH},
+    {"string", ACL_CATEGORY_STRING},
+    {"bitmap", ACL_CATEGORY_BITMAP},
+    {"hyperloglog", ACL_CATEGORY_HYPERLOGLOG},
+    {"geo", ACL_CATEGORY_GEO},
+    {"stream", ACL_CATEGORY_STREAM},
+    {"pubsub", ACL_CATEGORY_PUBSUB},
+    {"admin", ACL_CATEGORY_ADMIN},
+    {"fast", ACL_CATEGORY_FAST},
+    {"slow", ACL_CATEGORY_SLOW},
+    {"blocking", ACL_CATEGORY_BLOCKING},
+    {"dangerous", ACL_CATEGORY_DANGEROUS},
+    {"connection", ACL_CATEGORY_CONNECTION},
+    {"transaction", ACL_CATEGORY_TRANSACTION},
+    {"scripting", ACL_CATEGORY_SCRIPTING},
```

Comment:
for a moment i thought that maybe should avoid renaming these to protect the blame log,
but since we also created a new variable for them `acl_categories`, i suppose renaming is better, otherwise we risk them being check against the wrong flag somewhere (other branches).